### PR TITLE
Adds deploy to CI

### PR DESCRIPTION
**Pull request by lgehr**
as discussed

### CODE REVIEW

**Comment by lgehr** [comment]
I think the CI only fails, because it can not create tags in this environment. We should merge 
(also it works on my repo...)

**Review by mrd0ll4r — APPROVED** [review]

File: .github/workflows/Build_release.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    branches:
+    - "master"
+jobs:
+  build:
+    name: Build & Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup
```
Comment (mrd0ll4r) [inline .github/workflows/Build_release.yaml:15]: I guess we don't actually need Go outside of the container, right?

File: .github/workflows/Lint_Build.yaml [hunk 2]
```diff
@@ -45,8 +45,6 @@ jobs:
         fi
     - name: Build
       run: go build -v -buildmode=plugin -o mexport.so
-    - name: Build Docker Image
```
Comment (mrd0ll4r) [inline .github/workflows/Lint_Build.yaml:48]: Let's leave this in here, I want this tested on PRs :)

**Review by mrd0ll4r — CHANGES_REQUESTED** [review]

**Review by mrd0ll4r — APPROVED** [review]
LGMT, ship it!

**Pull request merged**